src/client/delphi_epidata.py

```diff
@@ -71,9 +78,19 @@ def _list(values):
     def _request_with_retry(endpoint, params={}):
         """Make request with a retry if an exception is thrown."""
         request_url = f"{Epidata.BASE_URL}/{endpoint}/"
+        if Epidata.debug:
+            Epidata.logger.info(f"Sending GET request to URL: {request_url} | params: {params} | headers: {_HEADERS}")
```

Comment:
should probably log `Epidata.auth` here too since its used in the request

Reply:
suggest(non-blocking): since we're using [structlog](https://www.structlog.org/en/stable/getting-started.html#building-a-context), you can feed params in like a dict and it will handle the formatting, like
```py
Epidata.logger.info("Sending GET request", url=request_url, params=params, headers=_HEADERS)
```